matrix/common/request/request_tracker.py

```diff
@@ -209,8 +245,24 @@ def log_error(self, message: str):
         :param message: str The error message to log
         """
         logger.debug(message)
-        self.dynamo_handler.write_request_error(self.request_id, message)
+        self.dynamo_handler.set_table_field_with_value(DynamoTable.OUTPUT_TABLE,
+                                                       self.request_id,
+                                                       OutputTableField.ERROR_MESSAGE,
+                                                       message)
         self.cloudwatch_handler.put_metric_data(
             metric_name=MetricName.REQUEST_ERROR,
             metric_value=1
         )
+
+    def write_batch_job_id_to_db(self, batch_job_id: str):
+        """
+        Logs the batch job id for matrix conversion to state table
+        """
+        self.dynamo_handler.set_table_field_with_value(DynamoTable.STATE_TABLE,
+                                                       self.request_id,
+                                                       StateTableField.BATCH_JOB_ID,
+                                                       batch_job_id)
+        self.cloudwatch_handler.put_metric_data(
```

Comment:
Does this make more sense here or in `batch_handler.schedule_matrix_conversion`?

Reply:
changing